core/src/test/scala/anorm/H2Database.scala

```diff
@@ -5,11 +5,10 @@ import scala.util.Random
 
 trait H2Database {
 
-  def withH2Database[R](block: Connection => R) = {
-    val dbName = "test" + Random.alphanumeric.take(6).mkString("")
+  def withH2Database[R](block: Connection => R): R = {
+    val url = "jdbc:h2:mem:test" + Random.alphanumeric.take(6).mkString("")
+    val connection = DriverManager.getConnection(url, "sa", "")
 
-    Class.forName("org.h2.Driver")
```

Comment:
I would have thought that without this, the driver might not be loaded, and so won't have had a chance to register itself with DriverManager?  Tests are passing, so I guess I'm wrong, but why?